.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
[ ] your `.gitignore` file should contain a `node_modules` and a `test.md` file.kindly fix that 🤓.

it's on the project requirements: 👇
![ignore](https://user-images.githubusercontent.com/62156607/197749003-fb18736b-f6bb-42ad-96bb-050e4baa3076.PNG)

Reply:
I'm done adding test.md file